src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,11 @@
-export const PostList = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts }) => (
+  <>
+    <div className="PostList">
+      {posts.map(post => (
```

Comment:
Add error handling for the case when 'posts' is null or empty. The .map function will throw an error if 'posts' is undefined.